src/NuGet.Core/NuGet.ProjectManagement/Projects/FolderNuGetProject.cs

```diff
@@ -109,12 +103,12 @@ public override Task<IEnumerable<PackageReference>> GetInstalledPackagesAsync(Ca
                     await PackageExtractor.ExtractPackageAsync(
                         downloadResourceResult.PackageStream,
                         PackagePathResolver,
-                        nuGetProjectContext.PackageExtractionContext,
-                        PackageSaveMode,
+                        nuGetProjectContext.PackageExtractionContext ?? new PackageExtractionContext(),
```

Comment:
@yishaigalatzer,

I remember why I did this. INuGetProjectContext is an interface. And, setting the default value for PackageExtractionContext needs to happen for every implementation of INuGetProjectContext. Note that there multiple properties of INuGetProjectContext which are null by default.